ModernWallet/Classes/ViewController/Settings/BackupPrivateKey/BackupPrivateKeyStartViewController.swift

```diff
@@ -27,7 +27,15 @@ class BackupPrivateKeyStartViewController: UIViewController {
         startButton.setTitle(Localize("backup.newDesign.readyToWrite"), for: .normal)
         
         startButton.rx.tap
-            .flatMap { return PinViewController.presentPinViewController(from: self, title: Localize("newDesign.enterPin"), isTouchIdEnabled: false) }
+            .flatMapLatest { [weak self] _ -> Observable<Void> in
+                guard let strongSelf = self else { return Observable.just(()) }
```

Comment:
In my opinion, you should return ```.never()``` instead of ```.just()```. This is not a flow, but rather guideline.